package.json

```diff
@@ -116,6 +116,7 @@
     "postcss-cssnext": "^2.7.0",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0",
+    "sweetalert": "^1.1.3",
```

Comment:
Agreed, I npm installed sweetalert first thing before I even dug into the code, so I didn't realize how the existing libraries were structured. Better for consistency, I agree.